CHANGELOG.md

```diff
@@ -3,6 +3,69 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.36
+
+_August 5, 2021_
+
+Big thanks to the 4 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Polish the cell editing API (#2220) @m4theushw
+- ⚡️ Add `details` param to each callback option in `XGrid` (#2236) @DanailH
+- 💅 Work on internal optimizations and code separation (#2176, #2243, #2235, #2213) @flaviendelangle
+- 🐞 Fix one bug related to filtering
+
+### `@material-ui/data-grid@v4.0.0-alpha.36` / `@material-ui/x-grid@v4.0.0-alpha.36`
+
+#### Breaking changes
+
+- [DataGrid] Polish cell editing (#2220) @m4theushw
+
+  - Replace `onCellModeChange` prop with `onCellEditEnter` or `onCellEditExit`.
```

Comment:
I think we should use the new callback names here

`
```suggestion
  - Replace `onCellModeChange` prop with `onCellEditStart` or `onCellEditStop`.
```

Reply:
Not sure. I would keep it as it was explained in the PR description

Reply:
`onCellEditEnter` does no longer exist

Reply:
I have updated https://github.com/mui-org/material-ui-x/pull/2220 to fix it there too